Userland/Applications/PixelPaint/Tools/TextTool.cpp

```diff
@@ -30,6 +30,11 @@ void TextToolEditor::handle_keyevent(Badge<TextTool>, GUI::KeyEvent& event)
     TextEditor::keydown_event(event);
 }
 
+Span<NonnullRefPtr<GUI::Action>> TextToolEditor::actions()
```

Comment:
This patch looks good, except that we normally return a `const&` to the vector type in situations like this instead of a span.

```
NonnullRefPtrVector<GUI::Action> const& TextToolEditor::actions()
{
    return m_actions;
}
```

Reply:
I'm not able to return a `const&` in this case, as the elements of the vector need to call a non-const function. My intention when returning the `Span` was to return a read-only view of the Vector while still being able to mutate its elements. I'm not sure of the most idiomatic way to achieve this.

I've changed what I had to return a shallow copy of the vector instead, as this seems to be what is done elsewhere.